packages/lambda-xyz/src/cli/serve.ts

```diff
@@ -95,9 +100,46 @@ async function main(): Promise<void> {
         }
     });
 
+    app.get(
+        '/v1/tiles/:imageryName/:projection/WMTSCapabilities.xml',
+        async (req: express.Request, res: express.Response) => {
+            const startTime = Date.now();
+            const requestId = ulid.ulid();
+            const logger = LogConfig.get().child({ id: requestId });
+            const { imageryName, projection } = req.params;
+            const ctx = new LambdaContext(
+                {
+                    httpMethod: 'get',
+                    path: `/v1/tiles/${imageryName}/${projection}/WMTSCapabilities.xml`,
+                } as any,
+                logger,
+            );
+            try {
+                const data = await lambda.handleRequest(ctx);
```

Comment:
IS this the same as above? could we make a function `await runLambda(path, req, res)` which does all the try catch/res.end stuff